test.js

```diff
@@ -4,5 +4,13 @@ var fileUrl = require('./');
 
 it('should convert path to file url', function () {
 	assert(/file:\/\/\/.*\/test\.jpg/.test(fileUrl('test.jpg')));
-	assert.strictEqual(fileUrl('/Users/sindresorhus/dev/te^st.jpg'), 'file:///Users/sindresorhus/dev/te%5Est.jpg');
+	if (process.platform === 'win32') {
```

Comment:
Well, at the moment you are assuming everyone has this repo cloned in a user directory named `sindresorhus`. I think that is a much bigger assumption ;)